interfaces/builtin/classic_support_test.go

```diff
@@ -81,8 +88,10 @@ func (s *ClassicSupportInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	c.Check(string(snippet), testutil.Contains, "/usr/bin/systemd-run Uxr,\n")
 	c.Check(string(snippet), testutil.Contains, "/bin/systemctl Uxr,\n")
 	// connected plugs have a non-nil security snippet for seccomp
-	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecuritySecComp)
+	seccompSpec := &seccomp.Specification{}
```

Comment:
That's a good point. I'll address it in another branch with a proposal of a helper + one of the tests updated to use it to mimimize the initial diff, if that's ok with you?